README.md

```diff
@@ -1,90 +1,130 @@
+
+Learn more about our values and our mission: [Flashbots - Frontrunning the MEV Crisis](https://medium.com/flashbots/frontrunning-the-mev-crisis-40629a613752)
+<br> Join us on Discord: [#Flashbots](https://discord.gg/7hvTycdNcK)
 
-## Flashbots Proof-of-concept (PoC)
-</br> We have designed and implemented a proof of concept for permissionless MEV extraction called MEV-Geth. It is a sealed-bid block space auction mechanism for communicating transaction order preference. While our proof of concept has incomplete trust guarantees, we believe it's a significant improvement over the status quo. Find out more about Flashbots architecture and implementation in our [ethresearch post](https://ethresear.ch/t/flashbots-frontrunning-the-mev-crisis/8251).
+### Index:
+* [Flashbots Alpha]()
+* [MEV-Explore]()
+* [Our organization]()
+* [Interested in contributing?]()
+* [MEV Roasts recordings]()
+* [List of Resources]()
+* [Navigate Flashbots' Github]()
 
+----
 ## Flashbots Alpha
 
-Flashbots Alpha is an early stage implementation of a direct communication channel between miners and Ethereum users that would like to communicate their preferred transaction order within a block.
+On Jan 6th 2021, we entered the Flashbots Alpha phase by opening up a proof-of-concept communication channel between traders and miners. This proof-of-concept is made of two components: MEV-Geth, a slightly modified fork of the Ethereum Geth client, and MEV-Relay, a miner relayer.
 
-Instead of broadcasting to the peer-to-peer network and competing with all Ethereum users, Flashbots users bid against each other in providing a reward directly to miners through a seal-bid auction. DeFi traders such as liquidators and arbitrageurs, can use Flashbots to bid for transaction inclusion without paying any gas fee for failed transactions, nor do they need to worry about leaking strategies before their trade was mined on-chain. 
+**Alpha flow:**
+* A trader sends a Flashbots bundle to MEV-Relay. The bundle contains:
+  * one or several transactions that can be the trader's and/or pending transactions in the mempool
+  * a sealed tip that is paid to the miner
+* MEV-Relay receives this bundle and sends it to all whitelisted MEV-Geth miners
+* Miners receive Flashbots bundles through MEV-Geth. 
+  * there can only be a single bundle per block
+  * the Flashbots bundle will always be at the top slot of the block by design of the system.
+* MEV-Geth picks which bundle to choose for each block.
+  * MEV-Geth will pick the most profitable bundle for the miner.
+  * MEV-Geth will compare the block that includes the bundle vs a vanilla Geth block, and will default back to the vanilla Geth block in case it is more profitable.
+* Only when the trader's bundle is included in a block do they pay the tip associated to their bundle.
```

Comment:
...block do they pay.. -> block they do pay

Reply:
changed the language to be a lil clearer